.github/workflows/stale-pr.yml

```diff
@@ -0,0 +1,43 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    # Execute every hour at xx:05 to avoid conflicts with other workflows
+    - cron: '5 * * * *'
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          operations-per-run: 50
+          ascending: true
+          exempt-all-milestones: true
+
+          # ISSUES (deactivated) ----------------------------------------------
+          # This workflow should touch no issues, so times are set to -1
+          # (see actions/stale documentation for the behavior)
+          days-before-issue-stale: -1
+          stale-issue-label: stale
+          stale-issue-message: >
+            **BUG!** This issue should not be marked stale by the "stale" workflow.
+            Please report it to @bt-developer-productivity
```

Comment:
❓ Should reference `@bt-support` instead

Reply:
Right, it was taken from the old template